apps/web/cypress/tests/integrations-list-page.spec.ts

```diff
@@ -1,3 +1,12 @@
+// @ts-nocheck
+
+Cypress.on('window:before:load', (win) => {
+  win._cypress = {
+    ...win._cypress,
+    IS_INTEGRATIONS_LIST_PAGE_ENABLED: 'true',
+  };
+});
```

Comment:
similar approach with flag `true`, this way we can run both e2e tests for these pages